CHANGES.md

```diff
@@ -3,6 +3,10 @@
 This document describes the relevant changes between releases of the OCM API
 SDK.
 
+## 0.1.399
+- [OSDEV-1296] Remove StatusBoard `fullname` search parameter.
+- Model updated to version v0.0.352.
```

Comment:
Bumping the model has a certain syntax for it, see other examples in this file. Also note that 0.0.352 includes other changes other than OSDEV-1296

Reply:
Hi @tzvatot, Thank you so much for the feedback. I modified the syntax and added an entry for Deprecating the `Notify` resource.  Is there anything else I should change?

Reply:
@tinaafitz the entry should be below the model update line. I've linked an example. Please see other example when documenting bump of a model version.